control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -468,7 +468,7 @@ func (r *HostedControlPlaneReconciler) Reconcile(ctx context.Context, req ctrl.R
 		meta.SetStatusCondition(&hostedControlPlane.Status.Conditions, newCondition)
 	}
 
-	// Reconcile Degraded status
```

Comment:
Just curious, what is the purpose of adding the `@todo` here?

Reply:
this will not be part of the PR, I will remove it with debug logs. I was just bookmarking the line for myself :D